ansible/roles/debops.postfix/tasks/ldap.yml

```diff
@@ -0,0 +1,59 @@
+---
+
+- name: Make sure 'postfix/ldap' directory exists
+  file:
+    dest: '/etc/postfix/ldap'
+    state: 'directory'
+    owner: 'root'
+    group: 'root'
+    mode: '0755'
+
+- name: Generate LDAP virtual files
+  template:
+    src:   'ldap/postfix__{{ item }}.j2'
+    dest:  '/etc/postfix/ldap/{{ item }}.cf'
+    owner: 'postfix'
+    group: 'postfix'
+    mode:  '0400'
+  loop:
+    - 'virtual_aliases'
+    - 'virtual_recipients'
+    - 'virtual_domains'
+    - 'virtual_senders'
+  no_log: '{{ postfix__no_log|d(True) }}'
```

Comment:
The `debops.postfix` role already implements and exposes a [mechanism for managing lookup tables](https://docs.debops.org/en/master/ansible/roles/debops.postfix/defaults-detailed.html#postfix-lookup-tables), you should use that instead of writing the lookup tables directly.

Reply:
Would that also work with a different role?

Reply:
Of course, that was the whole point to separate implementation details from other roles and focus them in `debops.postfix` role. Here's [definition of few lookup tables in `debops.postconf` role](https://github.com/debops/debops/blob/master/ansible/roles/debops.postconf/defaults/main.yml#L101-L181) and here's [how they are passed to the `debops.postfix` role](https://github.com/debops/debops/blob/master/ansible/roles/debops.postconf/defaults/main.yml#L214-L220). Note that you can still override the lookup tables via [these variables](https://github.com/debops/debops/blob/master/ansible/roles/debops.postconf/defaults/main.yml#L182-L201) separately from the `debops.postfix` role in the inventory. It's all done so that modifications through the Ansible inventory are possible.